### ISSUE TITLE
opam: "License doesn't adhere to the SPDX standard"

### ISSUE THREAD

**Issue by reynir** [paurkedal/ocaml-caqti#76]
When I run `opam install --with-test --with-doc --deps-only .` in latest master (46ffb8f840875da5ef67df51063443ff6ad2d2c1) I get warnings like below and it does not seem to install the test dependencies. Using opam 2.1.0.

```
[WARNING] Failed checks on caqti package definition from source at
          git+file:///home/reynir/workspace/ocaml-caqti#master:
  warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/:
              "LGPL-3.0-or-later WITH LGPL-3.0-linking-exception"
```

**Comment by reynir** [comment]
Ok, I had caqti* packages installed from opam, and once I removed that the above command installed `benchmark` which I was missing. I will keep this issue open nonetheless since the warnings still occur.

**Comment by paurkedal** [comment]
The warning was addressed by ocaml/opam#4768.  I added the `benchmark` dependency to the master branch very recently.
